tests/bug-1280.sil

```diff
@@ -2,7 +2,7 @@
 \font[filename=.fonts/NotoSerifCJK-Regular.ttc]
 \nofolios
 \neverindent
-\script{
+\begin{script}
```

Comment:
BTW, wouldn't we use `lua` rather than `script`? (That was my understanding seeing `lua` being added, but not so quite sure about it after all.)

Reply:
99% honestly? Because I haven't updated vim-sile with syntax highlighting for it yet and since we aren't throwing any deprecation warnings yet I am not in a hurry to switch. The other 1% is because this is an old test file I was just updating the style for readability not refactoring the API surface area.